src/plots/cartesian/dragbox.js

```diff
@@ -393,9 +393,7 @@ function makeDragBox(gd, plotinfo, x, y, w, h, ns, ew) {
             return;
         }
 
-        if(redrawTimer === null) {
-            clearSelect(zoomlayer);
-        }
+        clearSelect(zoomlayer);
```

Comment:
this fixes what I botched in https://github.com/plotly/plotly.js/commit/51c3128718ff61ccaad259a7ff65b76c06b2d2a3

Reply:
Oh wait, this worked fine in https://github.com/plotly/plotly.js/commit/51c3128718ff61ccaad259a7ff65b76c06b2d2a3

It is the optimization of https://github.com/plotly/plotly.js/commit/bb022818e7fe6c2c808ad440a633558398853325 that broke this (except during the very first scroll event),